src/main/webapp/app/exercises/text/shared/manual-text-selection/manual-text-selection.component.html

```diff
@@ -20,12 +20,7 @@
                 we then have to consume the click-intent by way of the (mouseup)
                 event.
             -->
-            <a
-                (mousedown)="$event.preventDefault()"
-                (mouseup)="$event.stopPropagation(); assessAction()"
-                class="indicator__cta-link"
-                jhiTranslate="artemisApp.textAssessment.editor.assess"
-            ></a>
+            <a (mousedown)="$event.preventDefault()" (mouseup)="$event.stopPropagation(); assessAction()" class="indicator__cta-link">{{ assessText }}</a>
```

Comment:
@Olimbia23 What was the issue with this?

Reply:
<img width="416" alt="Screenshot 2021-09-22 at 8 26 32 PM" src="https://user-images.githubusercontent.com/51077603/134400647-94ad6abb-4c33-4e7e-a0a0-5998f688cde3.png">

Reply:
Out of interest: Would the translation pipeline work here? If it does, I think this would be a better solution than the instant translation.

Reply:
> Pipeline didn't work either